### DISCUSSION TITLE
ERROR 419: Page Expired When Copying Backpack 6 Pro Project with Laravel 11 to a New Environment

### DISCUSSION THREAD

**Discussion by azeddine-hf** [Laravel-Backpack/community-forum#1181] [Q&A (Help)]
I'm experiencing a 419 Page Expired error after copying my Laravel 11 project (which uses Backpack 6 Pro) to another environment. The project works fine in the original environment, but after moving it, I cannot submit any forms and get the 419 error.

**Comment by karandatwani92** [comment]
Hey @azeddine-hf 

Make sure the `APP_URL` in your `.env` file is correctly pointing to the URL you use to access your application in the browser, for example: http:127.0.0.1:8000 or http://something.test
```
APP_URL=http://something.test
```
